package.json

```diff
@@ -41,6 +41,7 @@
         "bootstrap": "^5.0.2",
         "bootstrap-select": "^1.13.18",
         "bootstrap4": "npm:bootstrap@4.6.0",
+        "datejs": "^1.0.0-rc3",
```

Comment:
Instead of datejs, i'd recommend using https://date-fns.org/ which is more popular these days and is well maintained.

Reply:
Alright, will change it as soon as I get some time.